Bottom: 572fb2632bed8440d0b863935e1022cf9fb8cc18
Top:    39b2f461ba9d7888fa8ca5bccebb21f1007caa8f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-24 16:36:09 +0100

Refresh of hid-gtk-refactor-storage-seman

---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index 7ab56f0..1a10251 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -226,8 +226,8 @@ ghid_note_event_location (GdkEventButton * ev)
       event_x = ev->x;
       event_y = ev->y;
     }
-  gport->pcb_x = VIEW_X (event_x);
-  gport->pcb_y = VIEW_Y (event_y);
+  gport->pcb_x = EVENT_TO_PCB_X (event_x);
+  gport->pcb_y = EVENT_TO_PCB_Y (event_y);
 
   moved = MoveCrosshairAbsolute (gport->pcb_x, gport->pcb_y);
   if (moved)
@@ -737,8 +737,8 @@ ghid_port_window_leave_cb (GtkWidget * widget,
 	  w = ghid_port.width * gport->zoom;
 	  h = ghid_port.height * gport->zoom;
 
-	  x0 = VIEW_X (0);
-	  y0 = VIEW_Y (0);
+	  x0 = EVENT_TO_PCB_X (0);
+	  y0 = EVENT_TO_PCB_Y (0);
 	  ghid_get_coords (NULL, &x, &y);
 	  x -= x0;
 	  y -= y0;
diff --git a/src/hid/gtk/gui.h b/src/hid/gtk/gui.h
index 991757b..29a5407 100644
--- a/src/hid/gtk/gui.h
+++ b/src/hid/gtk/gui.h
@@ -68,8 +68,8 @@ extern int ghid_flip_x, ghid_flip_y;
 #define	DRAW_X(x)	(gint)((SIDE_X(x) - gport->view_x0) / gport->zoom)
 #define	DRAW_Y(y)	(gint)((SIDE_Y(y) - gport->view_y0) / gport->zoom)
 
-#define	VIEW_X(x)	SIDE_X((gint)((x) * gport->zoom + gport->view_x0))
-#define	VIEW_Y(y)	SIDE_Y((gint)((y) * gport->zoom + gport->view_y0))
+#define	EVENT_TO_PCB_X(x)	SIDE_X((gint)((x) * gport->zoom + gport->view_x0))
+#define	EVENT_TO_PCB_Y(y)	SIDE_Y((gint)((y) * gport->zoom + gport->view_y0))
 
 /*
  * Used to intercept "special" hotkeys that gtk doesn't usually pass
